src/main/java/com/google/cloud/genomics/dataflow/utils/ShardedBAMWritingOptions.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloud.genomics.dataflow.utils;
```

Comment:
I put the options into the DoFns, then created options classes in the transforms that extend all of their DoFn's options. So the pipeline creates an options class that extends the options from all the transforms it uses.